src/resources/2025/BloodCubicZirconia.ts

```diff
@@ -110,11 +110,32 @@ export function availableCasts(skill: Skill, statFloor: number): number {
  */
 export function castDownTo(skill: Skill, statFloor: number): boolean {
   if (!have() || !COSTS.get(skill)) return false;
-  let casts = availableCasts(skill, statFloor);
-  while (casts) {
-    if (!useSkill(skill, casts)) return false;
-    casts = availableCasts(skill, statFloor);
-  }
+
+  if (!cast(skill, availableCasts(skill, statFloor))) return false;
```

Comment:
I don't really remember why it was written the old way, but how concerned are we about mafia getting misaligned on a casts pref? we could do it as a one-at-a-time cast loop that repeatedly checks `availableCasts`, maybe? I guess this is strictly an improvement